### PR TITLE
Use twitter-typeahead instead of accessible-autocomplete

### PR DESCRIPTION
**Pull request by maxf**
Rationale: in order to make accessible typeahead work without JS, it has to be a dropdown.
We have hundreds of organisations and locations, so a dropdown would be unusable. Instead,
a-t lets us use a normal text input and progressively enhance it with typeahead

### CODE REVIEW

File: app/assets/javascripts/govuk.js [hunk 1]
```diff
@@ -24,27 +24,10 @@ $(document).ready(function () {
...
-    accessibleAutocomplete({
-      element: locationInput,
-      id: 'location',
-      name: locationInput.getAttribute('name'),
-      source: suggest_locations,
-      showAllValues: true, dropdownArrow: () => '',
-    })
-  }
 
+  addAutocomplete('#publisher', 'publisher', publisherTitles)
```
Comment (hannako) [inline app/assets/javascripts/govuk.js]: Do you think we should split this out as a module and put it in its own file in `app/assets/javascripts/`

File: app/assets/stylesheets/components.scss [hunk 2]
```diff
@@ -41,6 +41,45 @@
...
+    background: $page-colour;
+    overflow-x: hidden;
+    overflow-y: auto;
+    max-height: 150px;
+  }
+  .tt-dropdown-menu {
+    width: 100%;
+  }
+  .tt-highlight {
+    text-decoration: underline;
```
Comment (hannako) [inline app/assets/stylesheets/components.scss]: I think we are missing ` cursor: pointer;` here

**Comment by hannako** [comment]
Also need to add a line to the changelog

**Comment by maxf** [comment]
@hannako : all done, I think.

**Pull request merged**